sdk/@launchdarkly/observability-node/codegen.ts

```diff
@@ -0,0 +1,15 @@
+import { type CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+	schema: '../../../../backend/public-graph/graph/schema.graphqls',
+	documents: ['src/**/*.gql'],
+	generates: {
+		'./src/graph/generated/': {
+			preset: 'client',
+			config: {
+				documentMode: 'string',
```

Comment:
Generate just the documents and types. This doesn't depend on using a graphql library.